language-design.md

```diff
@@ -59,7 +96,19 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+The post on grayscale tells us that API design is an iterative process. Throughout the design of a language, there could be multiple ways of accomplishing a task. In the Verou article, several ways of representing grayscale are mentioned and users are asked to vote on which they find to be most intuitive. Users are also asked to leave comments justifying their opinions. The article is a good example that API design is a collaborative process, a dialogue between user and developer. 
+
+The commenters provide an insightful discussion that reflects their varied perspectives on the issue. Some argue in favor of the various methods proposed by Verou while others suggest entirely new forms of grayscale representation. This is where the domain of the language becomes important. Commenter x_addict suggests the following, 
+
+> how about luma, lightness, value, brightness, intensity or key? All taken from the print or photography worlds.” [Verou, 2014] 
+
+This form might be the most intuitive representation of grayscale if the language were to be used in print or photography software. However, in a different context, users may struggle to understand the format. In order to make the most successful decisions for a language, developers should take user feedback into account while simultaneously keeping in mind what they want their language to be.
 
+After some discussion, we agreed that the most intuitive way to represent grayscale (in our opinion) was an option proposed by a user in the comments. Commenter David Moore suggests, 
+
+> Why not grayscale(lightness [, alpha])? [Verou, 2014] 
+
+This option was not suggested by Verou and demonstrates the impact that users should have on the process of a well-designed API. Had the developer made a decision without first consulting users, they would not have opened themselves to potentially more intuitive possibilities. 
 
```

Comment:
I think that this is good point--users are the most in touch with the different features they need in development. While Verou (and developers) have the knowledge to start these conversations, they need people to participate in them.